components/Header/index.test.jsx

```diff
@@ -0,0 +1,54 @@
+import { render, screen } from '@testing-library/react';
+
+import { sidebarTestIds } from '@/components/Sidebar';
+import Header, { headerTestIds } from '.';
+
+import useUser from 'hooks/useUser';
+import useProfile from 'hooks/useProfile';
+
+jest.mock('hooks/useUser');
+jest.mock('hooks/useProfile');
+
+const mockUseProfile = {
+  name: 'john doe',
+  avatarUrl: 'avatar.me',
+};
+
+describe('Header', () => {
+  beforeEach(() => {
+    useUser.mockImplementation(() => jest.fn());
+    useProfile.mockImplementation(() => mockUseProfile);
+
+    render(<Header />);
+  });
+
+  it('renders a logo', () => {
+    const logo = screen.getByTestId(headerTestIds.logo);
+
+    expect(logo).toBeVisible();
+  });
+
+  it('renders the user avatar', () => {
+    const userAvatar = screen.getByTestId(headerTestIds.userAvatar);
+
+    expect(userAvatar).toBeVisible();
```

Comment:
Should we also test the `src` attribute of the picture to be equal to `mockUserProfile.avatarUrl`? :)

Reply:
sure, updated the `next/image` mock with `src` property in e85c5c2cd509d617ead2a2c5d8358977644d224d